.github/workflows/tests.yml

```diff
@@ -36,6 +36,12 @@ jobs:
           python-version: "3.8"
           sphinx: ">=4,<5"
 
+    continue-on-error: >-
```

Comment:
@chrisjsewell this is here to match the currently set up branch protection.

Reply:
I removed this as I think its not necessary to ignore any